content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md

```diff
@@ -77,31 +77,44 @@ charts:
 8. Confirm your changes by selecting **Confirm & Create**.
 
 
-9. Click on **Next** to review the cluster profile. Save the cluster profile.
+9. Click on **Add New Pack**.
 
 
-10. Navigate to the left **Main Menu** and select **Clusters**.
+10. Use the following information to find the Spectro Cluster Metrics pack.
+    - **Pack Type**: Monitoring
+    - **Registry**: Public Repo
+    - **Pack Name**: Spectro Cluster Metrics
+    - **Pack Version**: 3.3.X or newer.
+
+
+11. Use the default values. Confirm your changes by selecting **Confirm & Create**.
+
 
+12. Click on **Next** to review the cluster profile. Save the cluster profile.
 
-11. Click on **Add New Cluster**. Select **Deploy New Cluster**.
 
+13. Navigate to the left **Main Menu** and select **Clusters**.
 
-12. Pick the infrastructure provider you selected for the cluster profile you created earlier. 
 
+14. Click on **Add New Cluster**. Select **Deploy New Cluster**.
 
-13. Go ahead and assign a name to the host cluster and select the registered account you wish to deploy the host cluster to.  Click on **Next**.
 
+15. Pick the infrastructure provider you selected for the cluster profile you created earlier. 
 
-14. Pick a cluster profile for your cluster. Click on **Next**.
 
+16. Go ahead and assign a name to the host cluster and select the registered account you wish to deploy the host cluster to.  Click on **Next**.
```

Comment:
```suggestion
16. Assign a name to the host cluster and select the registered account that will deploy it.  Click on **Next**.
```

Reply:
Or "Assign a name to the host cluster and select the registered account to deploy it."